tests/paddle/test_paddle_autolog.py

```diff
@@ -0,0 +1,194 @@
+import pytest
+import paddle
+import mlflow
+import mlflow.paddle
+
+NUM_EPOCHS = 6
+
+pytestmark = pytest.mark.large
+
+
+@pytest.fixture
+def paddle_model():
+    mlflow.paddle.autolog()
+
+    paddle.set_default_dtype("float32")
+    train_dataset = paddle.text.datasets.UCIHousing(mode="train")
+    eval_dataset = paddle.text.datasets.UCIHousing(mode="test")
+
+    class UCIHousing(paddle.nn.Layer):
+        def __init__(self):
+            super(UCIHousing, self).__init__()
+            self.fc = paddle.nn.Linear(13, 1)
+
+        def forward(self, feature):  # pylint: disable=arguments-differ
+            pred = self.fc(feature)
+            return pred
+
+    model = paddle.Model(UCIHousing())
+    optim = paddle.optimizer.Adam(learning_rate=0.01, parameters=model.parameters())
+    model.prepare(optim, paddle.nn.MSELoss())
+
+    model.fit(train_dataset, eval_dataset, epochs=NUM_EPOCHS, batch_size=8, verbose=1)
+
+    client = mlflow.tracking.MlflowClient()
+    run = client.get_run(client.list_run_infos(experiment_id="0")[0].run_id)
+    return model, run
+
+
+@pytest.mark.parametrize("log_models", [True, False])
+def test_paddle_autolog_log_models_configuration(log_models):
+    mlflow.paddle.autolog(log_models=log_models)
+
+    paddle.set_default_dtype("float32")
```

Comment:
Do we need this line?

https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/set_default_dtype_en.html#set-default-dtype says:

> The default dtype is initially float32

Reply:
Yes, I deleted it